cypress/integration/user_registration.spec.js

```diff
@@ -90,23 +90,23 @@ context("user registration", () => {
     before(() => {
       // Clear everything again so transaction center is empty
       cy.nukeAllState();
-      cy.nukeCache();
       cy.createProjectWithFixture();
     });
 
-    it("shows the correct transaction details for confirmation", () => {
+    it("shows the correct transaction details for confirmation", async () => {
```

Comment:
We should never use `async` in cypress specs as all commands are async by default and Cypress manages the queueing:
https://docs.cypress.io/guides/core-concepts/introduction-to-cypress.html#Commands-Are-Asynchronous

Besides, there is no `await` in the body.